extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -26,6 +26,11 @@
   color: #fff !important;
   text-align: start !important;
   z-index: 100001 !important;
+  visibility: hidden !important;
+}
+
+.i-amphtml-story-bookend[active] {
+  visibility: initial !important;
 }
```

Comment:
I wonder if this needs a comment that it's just for a11y.
Other animations happen so it might be good to note that this is unrelated to visual changes.